lib/learn_web/client/event.rb

```diff
@@ -1,31 +0,0 @@
-require 'learn_web/client/event/submission'
-
-module LearnWeb
-  class Client
-    module Event
-      attr_reader :client
-
-      IRONBROKER_URL = 'http://ironbroker-v2.flatironschool.com'
-
-      def client
-        @client ||= Faraday.new(url: IRONBROKER_URL) do |faraday|
-          faraday.adapter Faraday.default_adapter
-        end
-      end
-
-      def submission_endpoint
-        '/e/learn_gem'
-      end
-
-      def submit_event(params)
```

Comment:
This method was called here: https://github.com/learn-co/learn-submit/blob/6e27c0b47f55a49c38e03f9a007e7d82760294a8/lib/learn_submit/submission.rb#L60

And will be removed in: https://github.com/learn-co/learn-submit/pull/12